### DISCUSSION TITLE
Does this mean I can't use cs.github right away

### DISCUSSION THREAD

**Discussion by shaozhihao031** [community/community#16291] [Other Feature Feedback, Questions, & Ideas]
Does this mean I can't use cs.github right awa
![截屏2022-05-10 上午11 22 23](https://user-images.githubusercontent.com/26448117/167536325-4762e77d-9b92-47bc-89d9-80c6d52d733a.png)

**Comment by colinwm** [comment]
After you sign up, you're put on the waitlist and don't have immediate access. 

We grant acces to people on the waitlist pretty often, so you likely won't have to wait too long!
